api/applications.js

```diff
@@ -268,5 +268,41 @@ module.exports = [
                 }
             }
         }
+    },
+    {
+        method: 'POST',
+        path: '/applications/{applicationId}/emailNotification',
+        handler: async (request, h) => {
+            const uow = await request.app.getNewUoW();
+            const logger = request.server.app.logger;
+            const applicationId = request.params.applicationId;
+            const addressee = request.payload.addressee;
+            const body = request.payload.body;
+
+            logger.debug(`Sending email notification to ${addressee}, with body: ${body}`);
+
+            try {
+                await emailService.sendNotificationEmail(uow, request, applicationId, addressee, body);
+                return true;
+            } catch(err) {
+                logger.error(err);
+                throw err;
+            }
+            return false;
+        },
+        options: {
+            auth: {
+                strategies: ['jwt', 'application-token']
+            },
```

Comment:
Add
```
plugins: {
    requiredPermissions: ['ViewOrganizations']
},
```
Here, because we don't want anyone with a core JWT to be able to send emails. `ViewOrganizations` is a sys admin permission, so Noah and I decided thats fine enough of a lockdown for now.